rfc5424/syslog_message.go

```diff
@@ -136,9 +151,14 @@ func (sm *SyslogMessage) SeverityMessage() *string {
 }
 
 // SeverityLevel returns the text level for the current severity value.
-func (sm *SyslogMessage) SeverityLevel() *string {
+func (sm *SyslogMessage) SeverityLevel(short bool) *string {
```

Comment:
I would rather this be SeverityLevelShort because at the call site this doesn't look good:


``` level := SeverityLevel(true)``` It's hard to know what it means.

Reply:
Ok